erpnext/selling/doctype/sales_order/sales_order.py

```diff
@@ -102,18 +102,13 @@ def validate_sales_mntc_quotation(self):
 	def validate_delivery_date(self):
 		if self.order_type == 'Sales' and not self.skip_delivery_note:
 			delivery_date_list = [d.delivery_date for d in self.get("items") if d.delivery_date]
-			max_delivery_date = max(delivery_date_list) if delivery_date_list else None
-			if not self.delivery_date:
-				self.delivery_date = max_delivery_date
 			if self.delivery_date:
 				for d in self.get("items"):
 					if not d.delivery_date:
 						d.delivery_date = self.delivery_date
```

Comment:
Just tried doing this from console, these lines are causing the child table values to not get updated